app/views/publisher_information/what_managing_editors_can_do.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :browser_title, t("publisher_information.what_managing_editors_can_do.title") %>
+<div class="govuk-grid-row">
+  <%= render partial: "publication_info_nav", locals: {current_page: managing_editors_path } %>
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l page-title"><%= t("publisher_information.what_managing_editors_can_do.title") %></h1>
```

Comment:
I know this pattern is not introduced with this PR, but ideally we don't add CSS classes only for testing purpose (e.g. `page-title`). I'd rather use `h1` as a selector since we must have one on every page.

Reply:
Good point.  I've added a change that removes the `page-title` class.